src/index.html

```diff
@@ -7,6 +7,324 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
```

Comment:
Don't add empty lines between parent and child, this decreases readability of the code.